Ladybird/Android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          xmlns:tools="http://schemas.android.com/tools"
+          android:installLocation="auto"
+          android:versionCode="001"
+          android:versionName="head">
+<supports-screens android:anyDensity="true"
+                  android:largeScreens="true"
+                  android:normalScreens="true"
+                  android:smallScreens="true"/>
+<application
+        android:dataExtractionRules="@xml/data_extraction_rules"
+        android:fullBackupContent="@xml/backup_rules"
+        android:icon="@mipmap/ic_launcher"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.Ladybird"
+        tools:targetApi="31"
+        android:hardwareAccelerated="true"
+        android:label="ladybird"
```

Comment:
Ladybird with a capital L, and actually maybe make that translatable?

Reply:
I see you already have that string, so:
```suggestion
        android:label="@string/app_name"
```
And the same thing on the activity below